app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/NoCollectionsMessageViewHolder.kt

```diff
@@ -5,59 +5,210 @@
+                start = 16.dp,
+                bottom = 16.dp
+            )
+    ) {
+        Column(
+            Modifier
+                .padding(all = 16.dp)
+                .fillMaxWidth()
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                SectionHeader(
+                    text = stringResource(R.string.collections_header),
+                    modifier = Modifier
+                        .weight(1f)
+                        .wrapContentHeight(align = Alignment.Top)
+                )
 
-        store.flowScoped(viewLifecycleOwner) { flow ->
-            flow.map { state -> state.normalTabs.size }
-                .ifChanged()
-                .collect { tabs ->
-                    binding.addTabsToCollectionsButton.isVisible = tabs > 0
+                IconButton(
+                    onClick = { onRemoveCollectionsPlaceholder() }
+                ) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_close),
+                        contentDescription = stringResource(
+                            id = R.string.remove_home_collection_placeholder_content_description
+                        ),
+                        modifier = Modifier.size(24.dp),
+                        tint = FirefoxTheme.colors.textPrimary
+                    )
                 }
+            }
+            PrimaryText(
+                text = stringResource(R.string.no_collections_description2),
+                modifier = Modifier.fillMaxWidth(),
+                fontSize = 14.sp
+            )
+            if (normalTabsState.value!!.isNotEmpty()) {
+
+                Spacer(modifier = Modifier.height(12.dp))
+
+                NoCollectionsButtonItem(
+                    buttonText = "Save tabs to collection",
+                    onClick = onAddTabsToCollectionTapped
+                )
+            }
         }
     }
+}
 
-    companion object {
-        const val LAYOUT_ID = R.layout.no_collections_message
+/**
+ * No Collections Tabs Button
+ *
+ * @param buttonText The error button's text and accessibility hint.
+ * @param onClick The lambda called when the button is clicked.
+ */
+@Composable
+fun NoCollectionsButtonItem(buttonText: String, onClick: () -> Unit) {
+    Button(
+        onClick = onClick,
+        modifier = Modifier
+            .clip(RoundedCornerShape(size = 4.dp))
+            .fillMaxWidth(),
+        elevation = ButtonDefaults.elevation(defaultElevation = 0.dp, pressedElevation = 0.dp),
+        colors = ButtonDefaults.outlinedButtonColors(backgroundColor = FirefoxTheme.colors.actionPrimary),
+    ) {
+        Icon(
+            painter = painterResource(R.drawable.ic_tab_collection),
+            contentDescription = null,
+            tint = FirefoxTheme.colors.textOnColorPrimary,
```

Comment:
@gabrielluong, should the Icon color be `iconOnColor`?

Reply:
I think so.

Reply:
PositiveButton style was being used earlier

`<style name="PositiveButton" parent="NeutralButton">
        <item name="backgroundTint">?actionPrimary</item>
        <item name="iconTint">?iconOnColor</item>
        <item name="android:textColor">?textActionPrimary</item>
    </style>`